code/modules/mob/living/living.dm

```diff
@@ -1,3 +1,23 @@
+/mob/living/New()
+	. = ..()
+	generate_static_overlay()
+	for(var/mob/living/silicon/robot/mommi/MoMMI in player_list)
+		if(MoMMI.keeper && !MoMMI.emagged && !MoMMI.syndicate)
+			if(MoMMI.static_choice in static_overlays)
+				MoMMI.static_overlays.Add(static_overlays[MoMMI.static_choice])
+				MoMMI.client.images.Add(static_overlays[MoMMI.static_choice])
+			else
+				MoMMI.static_overlays.Add(static_overlays["static"])
```

Comment:
```
/mob/living/silicon/robot/mommi/generate_static_overlay()
	return
```
Byond: 'Wrong value for list'

I solved this by adding && syndicateOverlays.len to the first (keeper, emagged, syndi) check.